service/resource_router.go

```diff
@@ -0,0 +1,33 @@
+package service
+
+import (
+	"context"
+
+	"github.com/giantswarm/kvm-operator/service/keyv2"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/operatorkit/framework"
+)
+
+// NewResourceRouter determines which resources are enabled based upon the
```

Comment:
Nitpick: this should start with `newResourceRouter ` as well.

Reply:
it used to happen when you do multi things at same time. Thanks